.circleci/config.yml

```diff
@@ -307,16 +307,15 @@ workflows:
             - win_build_38
```

Comment:
To be consistent, in lines 156-160 (out of scope, can't comment there) you could do something like:
```
git init
git checkout -b main
git add -A
git commit -m 'Demo build [ci skip]'
git remote add origin https://github.com/quantumblacklabs/kedro-viz.git
git push origin main:gh-pages --force
```

Reply:
oh good shout! Thanks :)